src/pages/Help.vue

```diff
@@ -49,7 +49,19 @@
                     Your current argument would be:
                     <code v-if="doorstopTarget.length > 0">{{ doorstopTarget }}</code>
                     <code v-else>These parameters will be available after installing BepInEx.</code>
+                    <br/>
                 </p>
+                <br/>
+                <template v-if="doorstopTarget.length > 0">
+                    <p>
+                        <button class="button" @click="copyDoorstopTargetToClipboard" v-if="!copyingDoorstopText">
+                            <i class="fas fa-clipboard" v-tooltip.left="'Drag to reorder'"></i>
```

Comment:
Says "Drag to reorder". Anyway, I don't see this or the "Copy launch arguments" tooltip at all?